src/JsonApiTestCase.php

```diff
@@ -37,12 +37,12 @@ protected function buildMatcher()
      * If statusCode is set, asserts that response has given status code.
      *
      * @param Response $response
-     * @param string|null $filename
-     * @param int|null $statusCode
+     * @param string $filename
+     * @param int $statusCode (optional)
```

Comment:
```suggestion
     * @param int $statusCode
```

And to be honest, I would prefer to remove all redundant doc blocks

Reply:
If you'd be using ECS in this project there would be a fixer for that. :D

Reply:
It's already there! #137 :)